### PR TITLE
Feedback + Team page

### PR DESCRIPTION
**Pull request by HappyJuliana**
Desktop:
image
image

Responsive on iPhone SE:
image
image
image
image
image

### CODE REVIEW

File: icons/facebook light-blue.svg
Comment (IvanCeban) [inline icons/facebook light-blue.svg:1]: no spaces in file names. At all. Never ever.

File: index.html [hunk 1]
```diff
@@ -220,54 +276,369 @@
...
+            padding: 0;
+            gap: 0;
+        }
+
+        .card__photo {
+            width: 325px;
+            height: 230px;
+
+            @media screen and (max-width: 768px) {
+                width: 285px;
```
Comment (IvanCeban) [inline index.html:435]: avoid fixed width

**Review by IvanCeban — APPROVED** [review]
Great job!

Just rename the images, please. No spaces in file names

File: icons/facebook light-blue.svg
Comment (HappyJuliana) [reply icons/facebook light-blue.svg:1]: Got it! Thank you!

File: index.html (hunk 1 above)
Comment (HappyJuliana) [reply index.html:435]: Is it possible to use "max-width" instead of "width" in this case?

**Pull request merged**